x-pack/plugins/exploratory_view/public/index.ts

```diff
@@ -39,7 +39,14 @@ export { APP_ROUTE as EXPLORATORY_VIEW_APP_URL } from './constants';
 
 export type { UXMetrics } from './components/shared/core_web_vitals';
 
-export { ExploratoryView } from './components/shared';
+export {
+  getCoreVitalsComponent,
+  ExploratoryView,
+  FieldValueSuggestions,
+  FieldValueSelection,
+  FilterValueLabel,
+  SelectableUrlList,
```

Comment:
these components belong in shared place in observability in my opinion. They are not part of exp view and are used in other places as well.

Reply:
These components are not used in any app other than Ux and Synthetics, and only in the context of using the Exploratory View feature.

Therefore it makes sense to house them inside the Exploratory View app. Alternatively, they could be part of some other package that Ux and Synthetics share. The Observability app will not use the Exploratory View functionality.